pkg/config/agentmanagement.go

```diff
@@ -0,0 +1,188 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/go-kit/log/level"
+	"github.com/grafana/agent/pkg/config/instrumentation"
+	"github.com/grafana/agent/pkg/server"
+	"github.com/prometheus/common/config"
+)
+
+type BasicAuth struct {
+	Username     string `yaml:"username"`
+	PasswordFile string `yaml:"password_file"`
+}
+
+type RemoteConfiguration struct {
+	Labels       []string `yaml:"labels"`
```

Comment:
I was trying to configure this and it took me a while to figure out how to do it. Why not do it more yaml-esque and do it something like this?

```
    labels:
      - team: team-a
      - hostname machine-1
```

Reply:
Ah yeah, this is a holdover from them being CLI flags. I'll fix this

Reply:
Should be resolved now, it now parses to a `map[string]string`, so specifying labels would look like:
```yaml
labels:
  team: "team-a"
  hostname: "machine-1"
```